thorlcr/master/thdemonserver.cpp

```diff
@@ -38,6 +38,11 @@ class DeMonServer : public CSimpleInterface, implements IDeMonServer
     unsigned reportRate;
     CIArrayOf<CGraphBase> activeGraphs;
     UnsignedArray graphStarts;
+    double thorMasterRate = 0.0;
+    double thorSlaveRate = 0.0;
+    unsigned costLimit = 0;
```

Comment:
unsigned __int64.  I suspect using stats_type or a new cost_type would be clearer.